HL2 mapscript improvement (#245)
GMLambda/Lambda: changelog.md

0.9.18 (in development)
- Improved: env_screenoverlay now works like behavior from HL2.

> - Improved: d1_trainstation_05: Breen now have cutscene.

Reviewer: ```suggestion
- Improved: d1_trainstation_05: Breen now plays the expected cutscene.
```

- Improved: d3_citadel_03: Reworked confiscation field area to make it process faster.
- Fixed: env_screenoverlay causing error on some unexpected cases.
- Fixed: Open settings menu and close it too fast will leave a slider bar at top of screen.

> - Fixed: d2_coast_10: Vehicle not correct removed after parking to the warehouse.

Reviewer: ```suggestion
- Fixed: d2_coast_10: Vehicle not being removed after parking it in the warehouse.
```

> - Fixed: d3_citadel_03: A lua error at round beginning and a softlock in rare case.

Reviewer: ```suggestion
- Fixed: d3_citadel_03: A lua error at the beginning of the round and a rare potential soft-lock .
```

> - Fixed: d3_breen_01: Breen get stuck with the chair, make he a bit slower in the cutscene.

Reviewer: ```suggestion
- Fixed: d3_breen_01: Breen can get stuck on the chair making the cutscene a bit slower.
```
